src/Workleap.DomainEventPropagation.Subscription.PullDelivery/EventPullerService.cs

```diff
@@ -69,6 +77,29 @@ private async Task StartReceivingEventsAsync(EventGridTopicSubscription eventGri
         }
     }
 
+    private async Task<EventProcessingStatus> HandleEventAsync(ICloudEventHandler cloudEventHandler, CloudEvent cloudEvent, CancellationToken stoppingToken)
+    {
+        try
+        {
+            await cloudEventHandler.HandleCloudEventAsync(cloudEvent, stoppingToken).ConfigureAwait(false);
+            return EventProcessingStatus.Handled;
+        }
+        catch (Exception ex)
+        {
+            switch (ex)
+            {
+                case EventDomainTypeNotRegisteredException:
+                case CloudEventSerializationException:
+                case EventDomainHandlerNotRegisteredException:
+                    this._logger.EventWillBeRejected(cloudEvent.Id, ex);
+                    return EventProcessingStatus.Rejected;
+                default:
+                    this._logger.EventWillBeReleased(cloudEvent.Id, ex);
```

Comment:
À la ligne 75 changer le `logger.CloudEventCannotBeReceived` pour `this._logger.CloudEventCannotBeReceived`. Aussi à cet endroit également on pourrait cacher des exceptions

Reply:
J'ai le changement mais ici, on voit que le logger message est au niveau de `warning` donc je ne pense pas qu'elle serait cachée?